audits/ux/no-plugins.js

```diff
@@ -0,0 +1,91 @@
+/**
+ * @license
+ * Copyright 2016 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+const Audit = require('../audit');
+
+class NoPlugins extends Audit {
+  /**
+   * @override
+   */
+  static get tags() {
+    return ['UX'];
+  }
+
+  /**
+   * @override
+   */
+  static get name() {
+    return 'no-plugins';
+  }
+
+  /**
+   * @override
+   */
+  static get description() {
+    return 'Does not use any plugins';
+  }
+
+  /**
+   * @param {!Artifacts} artifacts
+   * @return {!AuditResult}
+   */
+  static audit(artifacts) {
+    // Early exit if there are any iframes that aren't sandboxed.
+    if (!artifacts.sandboxedIframes) {
```

Comment:
do you want to add a `debugString` for these cases to let the user know why this failed?